decidim-core/lib/decidim/core/engine.rb

```diff
@@ -129,7 +129,7 @@ class Engine < ::Rails::Engine
         end
 
         Decidim.stats.register :processes_count, priority: StatsRegistry::HIGH_PRIORITY do |organization, start_at, end_at|
-          processes = ParticipatoryProcesses::OrganizationPrioritizedParticipatoryProcesses.new(organization)
+          processes = ParticipatoryProcesses::OrganizationPrioritizedParticipatoryProcesses.new(organization, nil)
```

Comment:
Please, make this second argument optional with a default value of `nil`, to reduce changes.